externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/ExecuteStatement.scala

```diff
@@ -31,6 +32,11 @@ class ExecuteStatement(
     override val shouldRunAsync: Boolean,
     queryTimeout: Long)
   extends HiveOperation(OperationType.EXECUTE_STATEMENT, session) {
+
+  private val operationLog: OperationLog = OperationLog.createOperationLog(session, getHandle)
```

Comment:
this is not necessary. this operation file will never be visit at read side.

I have already reimplemented the interface, we shall add a unit test for this to verify it's ok or not(which we shall find out why)

https://github.com/apache/incubator-kyuubi/blob/f22a14f1e50af6faad2a64902367af3f63a4ffc9/externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/operation/HiveOperationManager.scala#L96-L128

Reply:
@yaooqinn, I'm confusing about whether the `Log4j12DivertAppender` needs the operation. The operation log is used for setting the current operation log.